binary/core.py

```diff
@@ -169,41 +169,45 @@ def convert_units(
         except KeyError:
             raise ValueError(f'{to} is not a valid binary unit.')
 
+    babs = abs(b)
```

Comment:
Please check if this allows you to remove the type checking stuff:

```suggestion
    babs: Union[float, Decimal] = abs(b)
```

Reply:
That's what I tried initially but no, whatever my IDE uses for typing understands that it's a `float` or a `Decimal`, but `mypy` thinks it's an `object`, so it fails...
```
mypy: commands[0]> mypy .
binary/core.py:172: error: Incompatible types in assignment (expression has type "object", variable has type "float | Decimal")  [assignment]
        babs: Union[float, Decimal] = abs(b)
                                      ^~~~~~
Found 1 error in 1 file (checked 4 source files)
```

Reply:
Weird, then I guess you'll have to do `typing.cast(...)`

Reply:
I didn't know about that function ! Done, and the linter passes